client/src/components/pages/Subject/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Button from '../../common/Button';
+
+import('./index.css');
+
+const Subject = ({ subject = 'Arabic' }) => {
+  return (
+    <React.Fragment>
+      <div className="Student__div">
+        <p className="subject__p">{subject}</p>
+        <p className="subject__des">
```

Comment:
the above two basically follow the BEM convention, but I would still change them to `subject__name` and `subject__desc` (desc or descr is common abbreviation for description)